src/types/content-props/page-meta-props.ts

```diff
@@ -0,0 +1,6 @@
+import { ContentCommonProps, ContentType } from './_content-common';
+
+export type PageMetaProps = ContentCommonProps & {
+    type: ContentType.PageMeta;
+    data: any;
```

Comment:
Burde nok ha en type for denne, med undertyper som kan deles med v2 av sidetypene, f.eks:
```
    data: {
        contentType: OptionSetSingle<{
            product_page: ProductPageMetaData;
            situation_page: SituationPageMetaData;
            // <...osv>
        }>;
    };
```

Reply:
Ups, ja det må vi!